news/_posts/2020-03-12-ps17-minor-release-lifecycle.md

```diff
@@ -12,115 +12,118 @@ tags:
 
 For example 1.7.7.0 scope contained, but not only:
+
 - Compatibility with php 7.3
 - Migration and rework of Back-office Order pages
 - Advanced currencies management
 
-Although it is very hard to estimate the size of this scope, we try to size it in order for the development phase to last 4 months[2].
+Although it is very hard to estimate the size of this scope, we try to size it in order for the development phase to last 4 months[1].
+
+Once this scope has been is completed – i.e all issues have reached the "Done" column of the Kanban – the project reaches the [Feature Freeze](https://en.wikipedia.org/wiki/Freeze_(software_engineering)) stage.
 
-When this scope is completed - i.e all github issues have reached the "Done" column of the Kanban - we trigger the [Feature Freeze](https://en.wikipedia.org/wiki/Freeze_(software_engineering)).
+During this phase, no new items can be added to the version's scope, unless they are bugs related to code changes performed during the development of this version – called _regressions_. However, this is a _Feature Freeze_, not a _Code Freeze_, so some older bugs may be added to the scope if it is considered opportune to fix them quickly before the release is out (e.g. security fixes).
 
-We however accept that some issues remain in the Kanban, outside of "Done" column, only if they are bugs, no features. This is a Feature Freeze, not a Bug Freeze.
+## Feature Freeze
 
-# Feature Freeze
+Feature freeze means that all features of this version have been done and no new ones may be accepted in its scope. The project enters a phase of _stabilization_ whose aim is to identify and fix all bugs related to changes in the code during the development of this version (called _regressions_) before it's released.
 
-Feature freeze means that we freeze the development of the version and enter a phase of stabilization, which aim is naturally the release some time later.
+Once this phase is started, Core maintainers create a git branch from `develop` branch which will carry the work to be done until the release (for 1.7.7.0, the branch name is `1.7.7.x`). From this moment on, only bug fixes can be merged into this branch. Incidentally, this is also the branch where all future patch versions from this minor version will be developed on (hence the `.x` at the end).
 
-We create a git branch from `develop` branch which will carry the work to be done until the release (for 1.7.7.0, the branch name is `1.7.7.x`). From this moment, only bug fixes can be merged into this branch.
+Also when feature freeze is reached for `1.7.7.x` branch, it means than the next minor release (1.7.8) development phase starts on `develop` branch ! A minor version development actually starts (albeit slowly) precisely when the previous version enters feature freeze.
```

Comment:
Remember that there's no space before `;`, `:`, `!` and `?` in English.

```suggestion
Also, since stabilization is performed in a separate branch (`1.7.7.x` in our example), development for the next minor release (1.7.8) can start on the `develop` branch! This means that the development of any given minor version development actually starts (albeit slowly) precisely the moment the previous version enters feature freeze.
```